bin/test.sh

```diff
@@ -59,8 +65,12 @@ check_ping() {
     run ping -c 10 api.stripe.com
 }
 
-check_mtr() {
-    run mtr -n --report api.stripe.com
+check_route() {
+	  if command -v mtr 2>/dev/null; then
+      run mtr -n --report api.stripe.com
+		elif command -v traceroute 2>/dev/null; then
+      run traceroute -n -m 20 api.stripe.com
+		fi
```

Comment:
Good point. This always happens to me on filetypes I haven't configured :/  anyway, fixed